### DISCUSSION TITLE
toolchain.yaml &  .prototools

### DISCUSSION THREAD

**Discussion by tomdavidson** [moonrepo/moon#684] [Q&A] [answered]
Coming back to a project with moon and checking out the new developments - lots to be excited about!

Moon uses proto but not [.prototools](https://github.com/moonrepo/moon/blob/4a7f954aaf95d26ebbd5275cdf2fed8ca6ed1581/crates/core/config/src/toolchain/config.rs#L14) ?  proto can install go but moon can not (at least its not in the toolchain.json schema)?

So the repo is to have  toolchain.yaml &  .prototools and on initialization run `$ proto use && moon steup` ?

**Comment by tomdavidson** [comment]
Aah gesh, just saw  v0.26 roadmap #659  that well us proto.

**Reply by milesj** [reply]
Yeah it'll land on monday!

https://github.com/moonrepo/moon/pull/673

**Comment by milesj** [comment, marked as answer]
Landed in 0.26: https://moonrepo.dev/blog/v0.26#full-proto-toolchain-integration
